### PR TITLE
new icon: bun (original)

### LINKED ISSUE #1282: [ICON REQUEST]: bun
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Bun is a fast all-in-one JavaScript runtime

### Links and sources

[Website](https://bun.sh/)
[GitHub](https://github.com/oven-sh/bun)

### SVG Logo
Logo

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by AmreshSinha**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1282**

## Link to prove your SVG is correct and up-to-date.

https://github.com/oven-sh/bun/blob/main/packages/bun-landing/public/logo.svg

## Additional Comments
There was no `svg` of wordmark and plain type in their original repository.

### CODE REVIEW

**Comment by AmreshSinha** [comment]
Bump.

File: devicon.json [hunk 1]
```diff
@@ -1002,6 +1002,25 @@
...
+            "zig",
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
```
Comment (Snailedlt) [inline devicon.json:1019]: We only allow mono-colored icons as fonts. 
For more info check out [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards) and [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions) in our wiki :)
```suggestion
            "font": []
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
The icon itself looks good, but since it's multi-colored it can't be used as a font. So either remove `original` from the font array, or add a new mono-colored (plain or line) icon which can be used as a font :)

**Comment by Snailedlt** [comment]
@AmreshSinha Do you want to make a plain or line version of this icon?
You can follow this guide for making a line icon: https://github.com/devicons/devicon/issues/1306#issuecomment-1297625294

If you don't want to, that's okey too :) We can ask someone else

**Comment by AmreshSinha** [comment]
> @AmreshSinha Do you want to make a plain or line version of this icon?
> 
> You can follow this guide for making a line icon: https://github.com/devicons/devicon/issues/1306#issuecomment-1297625294
> 
> 
> 
> If you don't want to, that's okey too :) We can ask someone else 

I can't think of the main color in this main logo for the plain one. :/ It would be very helpful if anyone can co-author this pr with a plain or line version.

**Comment by Snailedlt** [comment]
@lunatic-fox or @kilianpaquier
Any of you wanna make some plain and line versions of this icon?

**Comment by kilianpaquier** [comment]
> @lunatic-fox or @kilianpaquier
> Any of you wanna make some plain and line versions of this icon?

Hello, I can take a look next Saturday

**Comment by kilianpaquier** [comment]
> > @lunatic-fox or @kilianpaquier
> > Any of you wanna make some plain and line versions of this icon?
> 
> Hello, I can take a look next Saturday

Hello, here's a proposition for the bun original icon.

bun-original (1)

**Comment by lunatic-fox** [comment]
Hi @kilianpaquier!
The version you're making actually is the `plain` one. 🙂
I thought in something that way:
> *It's not a SVG*

Like preserving the mouth contour and no fill.

---

> *PS: I don't know why, probably a bug from my side since it works fine in other programs, but when I try to visualize from browser this happens:*


> *This is just a thing that I want to share. Don't need to worry about it.* 😀

**Comment by kilianpaquier** [comment]
> Hi @kilianpaquier! The version you're making actually is the `plain` one. 🙂 I thought in something that way:
> 
> > _It's not a SVG_
> 
> 
> 
> Like preserving the mouth contour and no fill.
> > _PS: I don't know why, probably a bug from my side since it works fine in other programs, but when I try to visualize from browser this happens:_
> 
> 
> 
> > _This is just a thing that I want to share. Don't need to worry about it._ 😀

Hello ! Yes obviously I didn’t right the correct word 😂.

Thanks for the feedback, I see it with the right color on my phone, but I’m gonna take a look at the attributes just to be sure.

Great idea for the line icon, I shouldn’t take me long tonight to draft it.

**Comment by kilianpaquier** [comment]
Hello,

I fixed the plain with the fill attribute instead of style. Plus here's also the line icon.

bun-plain
bun-line

**Comment by lunatic-fox** [comment]
Pretty good! 🤩
I think the only thing that's needed is to keep the little tongue contour on `plain` version.

**Comment by Snailedlt** [comment]
@lunatic-fox feel free to add the plain and line icons to the PR. Like you said, the plain version needs the tongue added back in :)

**Review by lunatic-fox — APPROVED** [review]
Approved! ✔
Thank you for this contribution! 🚀

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**